source/aws_solutions/qnabot/cli/qnabot_cli_helper.py

```diff
@@ -0,0 +1,446 @@
+            str_lines = ""
+            for entry in str_file_contents["qna"]:
+                str_lines = str_lines + json.dumps(entry) + "\n"
+            return str_lines
+        except json.decoder.JSONDecodeError as err_exception:
+            return error_response(
+                error_code="",
+                message=err_exception.msg,
+                comments="There was an error reading the file "
+                + source_filename
+                + ". Check the file format and try again",
+                status="Error",
+                show_error=True,
+            )
+        except UnicodeDecodeError as err_exception:
+            return error_response(
+                error_code="",
+                message=err_exception.__doc__,
+                comments="There was an error reading the file "
+                + source_filename
+                + ". Check the file format and try again",
+                status="Error",
+                show_error=True,
+            )
+        except TypeError as err_exception:
+            return error_response(
+                error_code="",
+                message=err_exception.__doc__,
+                comments="There was an error reading the file "
+                + source_filename
+                + ". Check the file format and try again",
+                status="Error",
+                show_error=True,
+            )
+    except OSError as err_exception:
+        return error_response(
+            error_code=err_exception.errno,
+            message=err_exception.strerror,
+            comments="There was an error reading the file " + source_filename + ". Check the path and try again",
+            status="Error",
+            show_error=True,
+        )
+
+
+def convert_jsonl_to_json(str_file_contents: str):
+    """
+    Convert to JSON format
+    :param strFileContents: contents of JSON Lines file
+    :return: processed JSON string output
+    """
+
+    lines = str_file_contents.splitlines()
+    str_output = ""
+
+    for entry in lines:
+        if str_output != "":
+            str_output = str_output + ", " + entry
+        else:
+            str_output = str_output + entry
+
+    if str_output != "":
+        str_output = '{"qna": [' + str_output + "]}"
+
+    return str_output
+
+
+def error_response(error_code: str, message: str, comments: str, status: str, show_error: bool):
+    """
+    Error response
+    :param error_code: exception error code
+    :param message: exception error message
+    :param comments: exception comments
+    :param status: status to return in response
+    :return: response json
+    """
+
+    return_response = {"error_code": error_code, "error_message": message, "comments": comments, "status": status}
+    return_response = json.dumps(return_response, indent=4)
+
+    if show_error:
+        click.echo("[Error] " + str(error_code) + ": " + message + ". " + comments)
+        sys.exit(1)
```

Comment:
Similar to earlier comments, it would be better to return a response to caller and have them do the sys.exit(1). Particularly the qnabot_cli.py can do the sys.exit(1) as needed.